operator/internal/central/carotation/rotation.go

```diff
@@ -58,18 +69,28 @@ func DetermineAction(primary, secondary *x509.Certificate, current time.Time) Ac
 func Handle(action Action, fileMap types.SecretDataMap) error {
```

Comment:
I wonder if it makes sense to have this code in a separate package. It's only used in `reconcile_tls`, and if I moved this there, the coverage tool would stop complaining.